guides/openstack.md

```diff
@@ -146,6 +361,7 @@ public class JCloudsNova implements Closeable {
         String identity = "demo:demo"; // tenantName:userName
         String credential = "devstack";
 
+        // Please refer to 'Keystone v2-v3 authentication' chapter for complete authentication use case
```

Comment:
[minor] "section" rather than "chapter"? Also, should this comment perhaps come before the declaration of `String identity`?